portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/ExposeErcFromAssetLibraryForStructuredContents.testcase

```diff
@@ -59,11 +59,38 @@ definition {
 				title = "WC WebContent Title");
 		}
 
-		task ("Then structured Content is being createdAnd Then I can see the custom erc in the body response") {
+		task ("Then structured content is being created and I can see the custom erc in the body response") {
 			HeadlessWebcontentAPI.assertExternalReferenceCodeWithCorrectValue(
 				expectedactualExternalReferenceCodeValueValue = "erc",
 				responseToParse = "${response}");
 		}
 	}
 
+	@description = "StructuredContent is created in an asset library with default erc"
+	@priority = "5"
+	test StructuredContentIsCreatedInAssetLibraryWithErcByDefault {
+		property portal.acceptance = "true";
+
+		task ("When with POST request I create a structured Content without erc in asset library") {
+			var depotId = JSONGroupAPI._getDepotIdByName(depotName = "Test Depot Name");
+			var ddmStructureId = WebContentStructures.getDdmStructureId(structureName = "content-structure");
+
+			var response = HeadlessWebcontentAPI.createStructuredContentInAssetLibrary(
+				assetLibraryId = "${depotId}",
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "WC WebContent Title");
+		}
+
+		task ("Then structured content is being created and I can see erc equals to the key in the body response") {
```

Comment:
same here, **and** is a reserved word, please modify

Reply:
Updated in forced push.